src/DocumentDB.ChangeFeedProcessor/ChangeFeedProcessorBuilder.cs

```diff
@@ -109,6 +109,8 @@ namespace Microsoft.Azure.Documents.ChangeFeedProcessor
     /// </example>
     public class ChangeFeedProcessorBuilder
     {
+        internal const string IdPkPathName = "/" + DocumentServiceLease.IdPropertyName;
+        internal const string PartitionKeyPkPathName = "/" + DocumentServiceLease.LeasePartitionKeyPropertyName;
```

Comment:
This is changing the default of the PK, it would break any existing CFP running instances, wouldn't it? 

If I had a lease container created with `/id`, empty with no lease documents, and I start it, wouldn't the below check throw an exception because it does not match `/partitionKey`?